cmake/libc++.cmake

```diff
@@ -0,0 +1,5 @@
+# Specify this file as CMAKE_TOOLCHAIN_FILE when invoking CMake with Clang
```

Comment:
This file should be moved to the `.github` folder, since it's for interal use only in the CI.